app/util/appUtil.js

```diff
@@ -337,14 +337,36 @@ async function getItem(tokenId) {
 
   if (tokenId) {
     await api.isReady
-    const item = await api.query.simpleNftModule.tokensById(tokenId)
+    const rawItem = await api.query.simpleNftModule.tokensById(tokenId)
+    const item = rawItem.toJSON()
+    item.timestamp = await getTimestamp(item.created_at)
 
-    response = item.toJSON()
+    response = item
   }
 
   return response
 }
 
+async function getTimestamp(blockNumber) {
```

Comment:
Unit tests for this function?

Reply:
we mostly only do integration tests

Reply:
This would have to be refactored greatly for unit tests to be useful. Almost all the behaviour here is is tightly coupled to the object formats returned by the polkadot API so all you'd be doing by unit testing this specifically is moving the assumptions to the test infrastructure. Unit tests are most often useful when the function itself contains a lot of logic. 

We could refactor our code to separate out those concerns (which is probably what we should do), and then the logic could be unit tested. Given the relatively modest gain in this case I'm not too worried. The integration tests cover it and the logic is minimal so this is probably fine IMHO